libs/ui-components/src/components/Device/DeviceDetails/DeviceDetailsPage.tsx

```diff
@@ -70,7 +71,14 @@ const DeviceDetailsPage = ({ children, hideTerminal }: DeviceDetailsPageProps) =
       loading={loading}
       error={error}
       id={deviceId}
-      title={deviceAlias}
+      breadcrumbTitle={deviceAlias}
+      title={
+        canEdit ? (
+          <DeviceAliasEdit key={deviceAlias} deviceId={deviceId} alias={deviceAlias} onAliasEdited={refetch} />
```

Comment:
I think that the `key` is unnecessary ?
```suggestion
          <DeviceAliasEdit deviceId={deviceId} alias={deviceAlias} onAliasEdited={refetch} />
```

Reply:
Unfortunately, this was the only way I found to prevent a small but annoying bug.

With it, every time the pencil icon is clicked, the input field has the current alias value, and the cursor starts at the end. Only when there is no alias, we show "Untitled" as the placeholder.

Without it, the first time users click on the pencil, the input field shows the current value but only as the placeholder. Meaning if users want to append something to the current alias, they need to re-type it from the beginning.  Subsequent edits behave as expected.